apps/barista/src/environments/environment.ts

```diff
@@ -17,5 +17,5 @@
 export const environment = {
   deployUrl: '/',
   internal: true,
-  datahost: '',
+  dataHost: '',
```

Comment:
There is also an `environment.prod.ts` file, shouldn't it be updated too?

Reply:
If you rename this one here, did you change the integration on Jenkins for the replacement of this one as well? 
@lara-aigmueller ?